### PR TITLE
update kubeflow-pipeline and jupyter-web-app helm charts

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**

- Regenerate pipeline helm chart based on these two PR changes https://github.com/awslabs/kubeflow-manifests/pull/335 https://github.com/awslabs/kubeflow-manifests/pull/363 
- update installation path dict for kubeflow-pipeline  in vanilla and cognito 

**Testing:**
- [ ] Unit tests pass
- [x] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.istio.io/v1alpha3
```
Comment (surajkota) [inline charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml:1]: I think there is an issue in the srcipt, it pushed the jupyter web app related templates in kubeflow-pipelines in here

**Review by surajkota — CHANGES_REQUESTED** [review]
pipelines chart needs to be generated for all deployment options not only vanilla because of new KF server image and S3 related configuration

File: charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml (hunk 1 above)
Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml]: it was a mis-conduct when I was pasting the path, removed the jupyter web app content from pipeline chart. Actually I need to regenerate the chart for pipeline s3 only

Comment (jsitu777) [reply charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml]: done now @surajkota

Comment (surajkota) [reply charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml]: @jsitu777 what about rds-s3
https://github.com/awslabs/kubeflow-manifests/blob/main/charts/apps/kubeflow-pipelines/rds-s3/templates/ConfigMap/workflow-controller-configmap-kubeflow-ConfigMap.yaml#L11

Comment (surajkota) [reply charts/apps/kubeflow-pipelines/vanilla/templates/VirtualService/jupyter-web-app-jupyter-web-app-kubeflow-VirtualService.yaml]: @jsitu777 you need to regenerate chart for all kubeflow pipeline flavour to get in sync with
https://github.com/awslabs/kubeflow-manifests/pull/339/files#diff-402eb2df5dee333f2dc515c95eac04f19594757c7353a3feb15c8bf71aa89ff3R8-R10

**Review by surajkota — APPROVED** [review]

**Pull request merged**